dashboard/db/migrate/20171117004332_make_user_and_status_nullable_in_pd_applications.rb

```diff
@@ -0,0 +1,6 @@
+class MakeUserAndStatusNullableInPdApplications < ActiveRecord::Migration[5.0]
+  def change
+    change_column_null :pd_applications, :user_id, false
+    change_column_null :pd_applications, :status, false
```

Comment:
According to [this documentation](
https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_null), it looks like that last param should be `true`